src/components/workflowEditor/workflowEditor.tsx

```diff
@@ -44,154 +43,156 @@ class WorkflowEdit extends Component<WorkflowEditProps, WorkflowEditState> {
-                );
+                )
 
                 this.setState({
                     appName: result.appName,
                     workflows: result.workflows,
                     allCINodeMap: allCINodeMap,
                     allDeploymentNodeMap: allDeploymentNodeMap,
                     view: ViewType.FORM,
-                });
+                })
             })
             .catch((errors) => {
-                showError(errors);
-                this.setState({ view: ViewType.ERROR, code: errors.code });
-            });
-    };
+                showError(errors)
+                this.setState({ view: ViewType.ERROR, code: errors.code })
+            })
+    }
 
     getHostURLConfig() {
         getHostURLConfiguration()
             .then((response) => {
-                this.setState({ hostURLConfig: response.result });
+                this.setState({ hostURLConfig: response.result })
             })
-            .catch((error) => {});
+            .catch((error) => {})
     }
 
     showDeleteDialog = (workflowId: number) => {
-        this.setState({ workflowId, showDeleteDialog: true });
-    };
+        this.setState({ workflowId, showDeleteDialog: true })
+    }
 
     toggleCIMenu = (event) => {
-        let { top, left } = event.target.getBoundingClientRect();
-        top = top + 41;
+        let { top, left } = event.target.getBoundingClientRect()
+        top = top + 41
         this.setState({
             cIMenuPosition: {
                 top: top,
                 left: left,
             },
             showCIMenu: !this.state.showCIMenu,
-        });
-    };
+        })
+    }
 
     deleteWorkflow = () => {
         deleteWorkflow(this.props.match.params.appId, this.state.workflowId)
             .then((response) => {
                 if (response.status.toLowerCase() === 'ok') {
-                    this.setState({ showDeleteDialog: false });
-                    toast.success('Workflow Deleted');
-                    this.getWorkflows();
-                    this.props.getWorkflows();
+                    this.setState({ showDeleteDialog: false })
+                    toast.success('Workflow Deleted')
+                    this.getWorkflows()
+                    this.props.getWorkflows()
                 }
             })
             .catch((errors) => {
-                showError(errors);
-            });
-    };
+                showError(errors)
+            })
+    }
 
     handleCISelect = (workflowId: number | string, type: 'EXTERNAL-CI' | 'CI' | 'LINKED-CI') => {
-        let link = `${URLS.APP}/${this.props.match.params.appId}/edit/workflow/${workflowId}`;
+        let link = `${URLS.APP}/${this.props.match.params.appId}/edit/workflow/${workflowId}`
         switch (type) {
             case 'CI':
-                link = `${link}/ci-pipeline`;
-                break;
+                link = `${link}/ci-pipeline/0`
```

Comment:
is this intentional?

Reply:
Yes. We have introduced 1 more variable in the URL.

Reply:
ok